### ISSUE TITLE
val set of charades  dataset

### ISSUE THREAD

**Issue by ZHONGHZONG** [syuqings/video-paragraph#8]
Dear professor syuqings,

Thank you very much for your article,Sorry to bother you,Why is the val set of  charades dataset the same as the test set?
My assessment score was the same as the training model score.

I will be appreciated for your reply as soon as possible, thank you very much!

Best regards,

**Comment by syuqings** [comment]
Hi, the official split of Charades Captions dataset [Video captioning via hierarchical reinforcement learning] is 6963 for training, 500 for validation and 1760 for test. However, the downloaded annotations have 7473 items labeled as "train", and 1760 labeled as "test". And there is no video labeled as "validation". It seems that the training set and validation set have been merged. Therefore, we just validate the model on the 1760 testing set.

**Comment by ZHONGHZONG** [comment]
Hi，Thank you very much for your reply,Sorry to be a bother,Happy every day！
